subhub/api/webhooks/stripe/controller.py

```diff
@@ -11,6 +11,11 @@
 from subhub.api.webhooks.stripe.customer import StripeCustomerSubscriptionUpdated
 from subhub.api.webhooks.stripe.customer import StripeCustomerSubscriptionDeleted
 from subhub.api.webhooks.stripe.customer import StripeCustomerSourceExpiring
+from subhub.api.webhooks.stripe.invoices import (
```

Comment:
why not follow the pattern above, where you import one object and then another on the next line?  doing so would avoid this ugly parenthesized import

Reply:
addressed in update